firstvoices/backend/management/commands/convert_heic.py

```diff
@@ -127,6 +127,21 @@ def convert_images(self, images, logger):
                 )
                 continue
 
+    def log_orphaned_heic_images(self, site, logger):
+        orphaned_images = ImageFile.objects.filter(
+            Q(mimetype__iexact="image/heic")
+            | Q(content__iendswith=self.HEIC_EXTENSION),
+            site=site,
+        )
+
+        if orphaned_images:
+            logger.info(
+                f"The following orphaned HEIC images were found for site {site.slug}, and not converted:\n"
```

Comment:
since the log right before this says "no heic images found for this site", maybe this log could call them "orphaned heic files" to avoid confusion